Cargo.toml

```diff
@@ -30,4 +32,9 @@ tokio-stream = { version = "0.1" }
 tracing = "0.1.13"
 
 near-indexer-primitives = ">=0.16.0,<0.17.0"
-async-stream = "0.3.3"
+
+[dev-dependencies]
+aws-smithy-http = "0.53.0"
+
+[lib]
+doctest = false
```

Comment:
Since we are now running tests on PRs, i've disabled doc tests as they are failing. Although they should probably be fixed 😅

Reply:
Could you provide errors here so we can handle them? Or create a separate issue to do it later and provide errors there

Reply:
https://github.com/near/near-lake-framework-rs/issues/60